test-case/check-pause-resume-suspend-resume.sh

```diff
@@ -0,0 +1,113 @@
+#!/bin/bash
+
+##
+## Case Name: check-pause-resume-suspend-resume
+## Preconditions:
+##    N/A
+## Description:
+##    test playback with pause/resume as well as suspend/resume
+##    fake pause/resume with expect on aplay process
+##    when aplay is paused, have system enter suspend state for 5 secs
+##    resume from suspend state
+##    release aplay from paused state, then pause again
+##    repeat
+## Case step:
+##    1. aplay process is started
+##    2. aplay process is then paused via mock spacebar press via expect
+##    3. once aplay is paused, have system enter suspend state for 5 secs
+##    4. resume system from suspend state and unpause aplay process
+##    5. repeat from #2 - #4 50x
+## Expect result:
+##    no errors occur for either process
+##
+
+source $(dirname ${BASH_SOURCE[0]})/../case-lib/lib.sh
+
+OPT_OPT_lst['p']='pcm'          OPT_DESC_lst['p']='playback pcm. Example: hw:0,0'
+OPT_PARM_lst['p']=1             OPT_VALUE_lst['p']='hw:0,0'
+
+OPT_OPT_lst['l']='loop'         OPT_DESC_lst['l']='loop count'
+OPT_PARM_lst['l']=1             OPT_VALUE_lst['l']=5
+
+OPT_OPT_lst['i']='sleep-period' OPT_DESC_lst['i']='sleep period of aplay, unit is ms'
+OPT_PARM_lst['i']=1             OPT_VALUE_lst['i']='100'
+
+OPT_OPT_lst['s']='sof-logger'   OPT_DESC_lst['s']="Open sof-logger trace the data will store at $LOG_ROOT"
+OPT_PARM_lst['s']=0             OPT_VALUE_lst['s']=1
+
+func_opt_parse_option $*
+
+repeat_count=${OPT_VALUE_lst['l']}
+#TODO: file name salt for capture
+# configure random value range
+sleep_period=${OPT_VALUE_lst['i']}
+
+cmd=aplay
```

Comment:
can we also use param to get aplay or arecord here?

Reply:
yeah, thats a good idea.

Reply:
done.